assignments/week_1/exercise_1_2/exercise_1_2.cpp

```diff
@@ -0,0 +1,34 @@
+// Iterative C++ program to reverse an array and check if string is palindrome
+#include <stdio.h>
+#include <string.h>
+#include <iostream>
+
+int main()
+{
+    // Part 1: Array Reversal
+    int arr[] = {1, 2, 3, 4, 5, 6};
```

Comment:
If we don't want to teach `std::array`, we should stick to `std::vector`.
Codecademy doesn't actually cover language-level arrays... We should discuss as a group for real which containers we want to teach.